Config/AppConfiguration.swift

```diff
@@ -28,12 +28,19 @@ class AppConfiguration: CommonConfiguration {
     
     private func setupAppSettings() {
         // Enable CallKit for app
-        MXKAppSettings.standard()?.isCallKitEnabled = true
+        //MXKAppSettings.standard()?.isCallKitEnabled = true
```

Comment:
why not set `false` here?

we may add a comment about Tchap here BTW

Reply:
I restored the code we had in the duplicate file, but I agree : we might set it to false instead of commenting this line.